src/components/form/FieldHint.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+	Tooltip,
+	TooltipContent,
+	TooltipProvider,
+	TooltipTrigger
+} from "@zenml-io/react-component-library";
+import Info from "@/assets/icons/info.svg?react";
+import { JSONSchemaDefinition } from "@/types/forms";
+
+type Props = {
+	schema: JSONSchemaDefinition;
+	fieldName: string;
+};
+
+function getTypeHint(schema: JSONSchemaDefinition): string {
```

Comment:
There is a logic for auto generating a description, in case the backend doesnt provide one.

Do you think we should rather go with this auto-generated one, or not show a tooltip if the backend is not sending it @znegrin ?